whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapService.java

```diff
@@ -428,7 +428,6 @@ private Response redirect(final String requestPath, final Query query) {
         try {
             uri = delegatedStatsService.getUriForRedirect(requestPath, query);
         } catch (WebApplicationException e) {
-            LOGGER.error(e.getMessage(), e);
             throw new RdapException("404 Not found", "Redirect URI not found", HttpStatus.NOT_FOUND_404);
```

Comment:
Can you put logger.debug instead I am not sure in what all cases it throws this exception